modules/bitgo/package.json

```diff
@@ -83,6 +83,7 @@
     "@bitgo/sdk-coin-xtz": "^1.4.10",
     "@bitgo/sdk-coin-zec": "^1.3.10",
     "@bitgo/sdk-core": "^5.0.0",
+    "@bitgo/sdk-evm": "^1.0.0",
```

Comment:
let's avoid adding this here until we're ready to release it.

Reply:
I won't be able to have unit test without adding this to `modules/bitgo`